build/js/intlTelInput.js

```diff
@@ -218,6 +220,15 @@ https://github.com/Bluefieldscom/intl-tel-input.git
                 this.countryList.removeClass("v-hide").addClass("hide");
                 // this is useful in lots of places
                 this.countryListItems = this.countryList.children(".country");
+                // create dropdownContainer markup
+                if (this.options.dropdownContainer) {
+                    this.dropdown = $("<div>", {
+                        "class": "intl-tel-input iti-container"
+                    });
+                    $("<div>", {
+                        "class": "flag-dropdown"
+                    }).appendTo(this.dropdown).append(this.countryList);
```

Comment:
this.countryList needs to be appended to a visible element at some point prior to opening the menu in order to properly determine this.dropdownHeight. It'll take some code rearrangement to not append this.countryList to this.flagsContainer when the markup is generated. Appending countryList will have to be moved to _setDropdownPosition.

The nice thing is, it doesn't actually add any additional load time, but I do agree we could probably optimize it a bit further. Although, appending it twice only adds about 20ms per input, so we might be in the realm of micro-optimization.